xonsh/environ.py

```diff
@@ -710,7 +711,7 @@ def locate_binary(name):
 
     try:
         return next(chain.from_iterable(yield_executables(directory, name) for
-                    directory in builtins.__xonsh_env__.get('PATH') if os.path.isdir(directory)))
+                    directory in ([_get_cwd()] + builtins.__xonsh_env__.get('PATH')) if os.path.isdir(directory)))
```

Comment:
This is a case where there is a difference between POSIX and Windows. Win users would probably expect to be able to run executables from the cwd.